PackageInfo.g

```diff
@@ -104,12 +104,15 @@ ArchiveURL := Concatenation( ~.SourceRepository.URL,
 ArchiveFormats := ".tar.gz",
 
 
+##  Optional:
 ##  Information about authors and maintainers is contained in the `Persons'
 ##  field which is a list of records, one record for each person; each 
 ##  person's record should be as per the following example: 
 ##  
 ##     rec(
-##     # these are compulsory, the strings can be encoded in UTF-8 or latin1,
+##     # 'LastName' is mandatory.
+##     # 'FirstNames' is optional from GAP 4.14 on, was mandatory before.
+##     # The strings can be encoded in UTF-8 or latin1,
```

Comment:
That's not really true anymore either: if the text is not UTF-8, it will be displayed incorrectly on e.g. the website and other places. Because there is no magic way for us to detect different encodings...

In practice I would say everything should use UTF-8 encoding.